mkosi/config.py

```diff
@@ -116,6 +116,11 @@ def extension(self) -> str:
             OutputFormat.uki:  ".efi",
         }.get(self, "")
 
+    def can_compress(self):
```

Comment:
This name isn't entirely right. UKI and ESP can compress, they're just not compressed at the outer level. Instead the inner contents are compressed. Not sure what a better name would be. `is_self_decompressing()`?

Reply:
For directory, of course the name `can_compress()` does make sense. So maybe we need two separate methods.

Reply:
I'm not sure if "directory" is "self decompressing". I changed the name to `use_outer_compression`.